docs/installation.md

```diff
@@ -28,6 +28,32 @@ Generate all the mocks for your project:
 
 	docker run -v "$PWD":/src -w /src vektra/mockery --all
 
+### pre-commit
+
+Add the following lines to your `.pre-commit-config.yaml` file
+
+```yaml
+repos:
+  - repo: https://github.com/ccoVeille/mockery
+    rev: v2
+    hooks:
+      - id: generate-mocks
+```
+
+!!! note
+
+    If the file is missing, you can simply create it with this content
+
+Then install the hook with the following command:
+
+```shell
+pre-commit autoupdate
```

Comment:
Ah, this is why you have it as `v2` above, calling for autoupdate will which bump the version to latest.  May be better to be explicit about the versioning system, and call for `pre-commit install` versus autoupdate.

Reply:
Indeed, I'm used to use autoupdate for everything.

I think "pre-commit install" would also bump it locally. I will make some tests.

Then I will update the documentation based on my tests